test/e2e/next-test-lib/fetch-polyfill.js

```diff
@@ -0,0 +1,8 @@
+if (!globalThis.fetch) {
+  const fetch = require('node-fetch')
+
+  globalThis.fetch = fetch
+  globalThis.Headers = fetch.Headers
+  globalThis.Request = fetch.Request
+  globalThis.Response = fetch.Response
+}
```

Comment:
this is used just in Next's e2e tests setup, everywhere else there is global fetch (`@netlify/blobs` require `fetch` to be either passed in or to be globally available)